.readthedocs.yml

```diff
@@ -6,29 +6,22 @@
 version: 2
 
 build:
-  os: ubuntu-22.04
+  os: "ubuntu-24.04"
   tools:
     python: "3.9"
-    nodejs: "19"
-  jobs:
-    post_checkout:
-      - git fetch --unshallow || true
-    pre_build:
-      - pip freeze
-      - python -m sphinx -WETan -j auto -D language=en -b linkcheck -d _build/doctrees docs/source _build/linkcheck
 
-# Build documentation in the docs/ directory with Sphinx
-sphinx:
-  builder: html
-  configuration: docs/source/conf.py
-  fail_on_warning: true
+  commands:
+    # Install dependencies
+    - pip install -e ".[docs]"
```

Comment:
This was the longest step, maybe we can try using `uv`. Not an issue for kedro, but it will be an issue for kedro-datasets

Reply:
hey yeah good spot, we can use uv here for sure 👍